### ISSUE TITLE
GPO | Homepage

### ISSUE THREAD

**Issue by mbbinh** [mozilla/policy-templates#161]
I have a problem, I want to set homepage for FF but policy is not working.
Server: 2012 Standard
Client: Windows 10 Pro/1803 with FF Quantum 60.0.2
![image](https://user-images.githubusercontent.com/39734390/41189582-5e7d6c9e-6bfa-11e8-8029-49ea45d418a2.png)

Then apply policy. In Client, check registry have value homepage.
![image](https://user-images.githubusercontent.com/39734390/41189617-d7807dca-6bfa-11e8-8232-a08f4a2e769b.png)

FF still not change homepage.
help me, pls!

**Comment by mbbinh** [comment]
![image](https://user-images.githubusercontent.com/39734390/41189686-0e3d78a8-6bfc-11e8-9897-16ef11fcba91.png)
I download lastest policy-temp for FF

**Comment by rosskoes05** [comment]
I have this same issue with the latest GPO.

**Comment by mkaply** [comment]
Are you using the Firefox ESR or the regular Firefox?

**Comment by mbbinh** [comment]
@mkaply 
![image](https://user-images.githubusercontent.com/39734390/41264616-cbd320a8-6e17-11e8-8d0e-2a2ea264bb6f.png)

**Comment by BINSERV** [comment]
Same problem here. 

Homepage in GPO is set up and locked, but FF ESR 60.0.2 won't take it.

**Comment by rosskoes05** [comment]
ESR as well - v60.0.2

**Comment by CLiX90** [comment]
Same here.
Firefox ESR 60.0.2 (64Bit)
Windows 10 Enterprise 1703

**Comment by mbbinh** [comment]
@mkaply
Can you help us ???

**Comment by CLiX90** [comment]
It works for me if I add https:// or http:// to the domain.
Without the protocol the policy is not working.

But it is not possible to set browser.startup.page to 0 via autoconfig if you apply the homepage gpo.
@mkaply Please check it out. Thanks!

**Comment by rosskoes05** [comment]
@CLiX90 Adding https:// or http:// fixed my issue.  Thanks!

**Comment by mbbinh** [comment]
@CLiX90 
I try but unsuccessful

**Comment by BINSERV** [comment]
@CLiX90 

Adding the protocol worked for us as well. Thank you very much, sir!

**Comment by gomorrahsend** [comment]
@CLiX90 Same problem. Adding the protocol didn't work for me. Using Windows Server 2016 running Windows 10 1709 clients.

**Comment by CLiX90** [comment]
@gomorrahsend @mbbinh  Which URL did you set?

**Comment by mkaply** [comment]
Is there possibly a space before or after the URL?

Are there any errors on the browser console?

You can get additional details by creating a string preference in about:config called "browser.policies.loglevel" and setting it to "debug"

**Comment by seanjones150** [comment]
I'm having the same issues. ESR 60.0.2 as well. I've checked to make sure there are no other characters in the URL. I'm also specifying the protocol and its still not working for me.

**Comment by jab-au** [comment]
I'm seeing the same issue with Firefox 61.0.1 (Rapid)
I have set the "URL for Homage" to the desired internal address including the protocol and checked the the do not allow changes. I have checked for odd spaces in my URL.

Same results as others have indicated, when I open Firefox I get the new tab screen.

The polices are applying for other options i.e. Always check if Firefox is your default browser.

Noted that browser.policies.loglevel does not exist in about:config
